bundles/org.openhab.binding.siemensrds/README.md

```diff
@@ -107,8 +107,8 @@ sitemap siemensrds label="Siemens RDS"
 {
 	Frame label="Heating" {
 		Text      item=Upstairs_RoomTemperature
-		Setpoint  item=Upstairs_TargetTemperature minValue=15 maxValue=30 step=1
-		Switch	  item=Upstairs_ThermostatAutoMode
+		Setpoint  item=Upstairs_TargetTemperature minValue=5 maxValue=30 step=0.5
+		Switch	    item=Upstairs_ThermostatAutoMode
```

Comment:
Something when wrong with the indentation here. Maybe it's also better to replace all tabs in this readme with spaces to avoid indentation issues.

Reply:
Ok, hopefully fixed now